learning_journal/templates/single-entry.html

```diff
@@ -1,29 +1,20 @@
-<!-- templates/home.html -->
-
 {% extends "base.html" %}
 {% block body %}
-
   <main id="content" class="container">
     <a href="/" type="button" class="btn btn-info btn-arrow-left">Home</a>
     <section id="single-journal" class="row">
-      <h2 class="text-center">This is a blog title</h2>
+      <h2 class="text-center">{{ journal.title }}</h2>
       <div class="col-sm-6">
-        <a class="btn btn-primary" href="/journal/12345/edit-entry">Edit</a>
+        <a class="btn btn-primary" href="/journal/{{ journal.id }}/edit-entry">Edit</a>
```

Comment:
Use `request.route_url`.  You can pass additional arguments that will be used to fill these placeholders.  Never hard-code another url.